webview/src/experiments/components/table/body/commits/CommitsButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styles from './styles.module.scss'
+import { Icon, IconValue } from '../../../../../shared/components/Icon'
+import Tooltip from '../../../../../shared/components/tooltip/Tooltip'
+export interface CommitsButtonProps {
+  icon: IconValue
+  moreOrLess: 'More' | 'Less'
+  action: () => void
+  disabled: boolean
+}
+
+export const CommitsButton: React.FC<CommitsButtonProps> = ({
+  icon,
+  moreOrLess,
+  action,
+  disabled
+}) => {
+  const text = `Show ${moreOrLess} Commits`
+  return (
+    <Tooltip content={<>{text}</>}>
+      <button
```

Comment:
Maybe it's just me and my bad eyesight 👓, but the icons look pretty small compared to other table elements/icons. Should we consider increasing the size  by a couple pixels?